INFO     backend.layers.business.business:business.py:517 {'message': 'ingesting dataset', 'collection_version_id': 60e800d6-e7d6-4406-83cb-1689f3aa474d, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:517 {'message': 'ingesting dataset', 'collection_version_id': 60e800d6-e7d6-4406-83cb-1689f3aa474d, 'url': None, 'current_dataset_version_id': b33aaa19-df12-4844-9e46-e9eb4ec2aebc}
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:26 Start create_artifact
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:process_logic.py:91 Uploaded [5e7ca436-1f07-4d23-b6c9-f56b9dc8761b/raw.h5ad] to s3://artifact_bucket/5e7ca436-1f07-4d23-b6c9-f56b9dc8761b/raw.h5ad
INFO     processing:process_logic.py:93 Updated database with raw_h5ad.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:logger.py:28 Complete create_artifact
INFO     processing:process_logic.py:45 Updating processing status
ERROR    processing:dataset_metadata_update.py:261 Cannot find labeled H5AD artifact uri for b33aaa19-df12-4844-9e46-e9eb4ec2aebc.
INFO     processing:process_logic.py:45 Updating processing status
INFO     processing:dataset_metadata_update.py:265 Main: Starting thread for rds update
INFO     processing:dataset_metadata_update.py:288 Main: Starting thread for cxg update
INFO     processing:process_logic.py:45 Updating processing status